CMakeLists.txt

```diff
@@ -136,17 +213,13 @@ macro(add_h3_executable name)
     # invoke built-in add_executable
     add_executable(${ARGV})
     if(TARGET ${name})
-        target_link_libraries(${name} h3.1)
-        if (M_LIB)
-            target_link_libraries(${name} ${M_LIB})
-        endif()
-        target_include_directories(${name} PUBLIC src/apps/applib/include)
+        target_link_libraries(${name} PUBLIC h3)
+        target_link_libraries(${name} PUBLIC ${M_LIB})
```

Comment:
I think there needs to be an `if (M_LIB)` guard due to platforms that don't have libm (WIndows).

Reply:
At first I though that would be the case. Then I thought, M_LIB should expand to be empty if not found so it will be a no-op in that case and fine. But I thought I finally reverted back to the if guard after realizing M_LIB actually gets set to libm-NOTFOUND in that case (only in CMake would that be a falsy if value, check the docs for more on that). So if it isn't there, I'll definitely put the if back where it was.